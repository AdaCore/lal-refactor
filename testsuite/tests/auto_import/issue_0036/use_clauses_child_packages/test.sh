# This test checks that we don't fully qualify already
# partially-qualified names
auto_import -P test.gpr --source src/main.adb --line 2 --column 12
