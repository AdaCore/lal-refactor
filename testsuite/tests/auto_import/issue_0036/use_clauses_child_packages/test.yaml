description:
  This test checks that we correctly handle use-clauses on child packages

driver: shell_script
