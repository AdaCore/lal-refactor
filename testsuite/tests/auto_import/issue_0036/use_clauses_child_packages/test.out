Available Imports:
Import: Parent.Child; Qualifier: Parent

Edits:
main.adb
1:1-1:1 with Parent.Child;

2:8-2:8 Parent.

